Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -1045,7 +1045,7 @@ namespace OpenInfraPlatform {
 								double openingAngle = calculateOpeningAngle(senseAgreement, startAngle, endAngle);
 
 								numSegments = GeomSettings()->getNumberOfSegmentsForTessellation(radiusMin, abs(openingAngle));

```

Comment:
I recommend taking the bigger radius to determine the number of segments - therefore taking the bigger amount from the two (bigger radius = denser tessellation).

Reply:
Resolved: b02d4d7